themes/src/main/resources/theme/base/login/template.ftl

```diff
@@ -29,9 +29,19 @@
             <script src="${url.resourcesPath}/${script}" type="text/javascript"></script>
         </#list>
     </#if>
+    <#if properties.asyncScripts?has_content>
```

Comment:
This allows a theme extending base (or another theme that extends base) to add scripts that should be loaded with the async or defer parameters